agent/05_agent_configure.sh

```diff
@@ -519,6 +549,8 @@ if [[ "${AGENT_PLATFORM_TYPE}" == "external" ]] || [[ "${AGENT_PLATFORM_TYPE}" =
   set_device_mfg worker $NUM_WORKERS ${AGENT_PLATFORM_TYPE} ${AGENT_PLATFORM_NAME}
 fi
 
+block_insecure_machine_config_server_port extraworker $NUM_EXTRA_WORKERS
```

Comment:
As mentioned before, I was expecting the port to be blocked on the control plane nodes, more than on the hosts to be added